addons/website/static/src/js/menu/edit.js

```diff
@@ -103,7 +151,18 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
         }
         this.editModeEnable = true;
 
-        this.wysiwyg = await this._createWysiwyg();
+        this.wysiwyg = await this._createWysiwyg({
+            plugins: [[this.JWEditorLib.OdooField]],
+            mode: {
+                id: 'edit',
+                rules: [
+                    {
+                        selector: [this.JWEditorLib.DividerNode],
+                        breakable: false,
+                    },
+                ],
+            },
```

Comment:
I think the mode should be encapsulated by the wysiwyg logic. The role of the class wysiwyg is to hide the specific of the editor. But this configuration is specific to our editor.

Reply:
What would you suggest then? Why do we want to hide the params? I need a different configuration for the edit wysiwyg than for the translate wysiwyg. You want to have a `createTranslator` and `createEditor` in `loader.js` then?